Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -60,7 +58,7 @@ select new
 
         // List all keys for a team
         [HttpGet]
-        [ProducesResponseType(typeof(IEnumerable<Key>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(IEnumerable<KeyInfo>), StatusCodes.Status200OK)]
         public async Task<IActionResult> List()
```

Comment:
@wongband This does return a key, but we actually cut out a bunch of stuff.

I'm thinking that this should have a new model, maybe call KeyPartial?
                key = new Key
                {
                    Id = r.Id,
                    Code = r.Code,
                    Name = r.Name,
                    Notes = r.Notes,
                    TeamId = r.TeamId,
                    Tags = r.Tags
                },
                id = r.Id,
                SpacesCount = r.SpacesCount,
                SerialsInUseCount = r.SerialsInUseCount,
                SerialsTotalCount = r.SerialsTotalCount

Reply:
Yea that works as well